doc/installation/system/inifile.rst

```diff
@@ -45,7 +45,8 @@ The file should contain the following contents::
    # PI_INIT_CHECK_HOOK = 'your.module.function'
    # PI_CSS = '/location/of/theme.css'
    # PI_UI_DEACTIVATED = True
-
+   # This is used to create a custom list of hash algorithms.
```

Comment:
this is a quite specific configuration option which is not mandatory. I think we should not put it in this example config even when it is commented out.

Reply:
fixed in 54e1e2f7b08297e5070bc3846df744a8a6f90e1f.